symplyphysics/laws/hydro/volume_of_the_submerged_body_part_from_body_density_and_liquid_density.py

```diff
@@ -0,0 +1,43 @@
+from sympy import (Eq, solve)
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+                           validate_output)
+
+# Description
+## If the body is on the surface of a liquid (floating), then only two forces act on it (Archimedes up and gravity down), which balance each other.
+
+## Law: V_sub / V = ⍴ / ⍴_liquid

```

Comment:
Add a note that density of body should be less than density of fluid. Otherwise it drowns and forces do not compensate each other.
Also mention that density is not density of material of the body, but density of its full volume, ie a sheet of metal in form of a vessel will float.